specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview/managedClusters.json

```diff
@@ -5820,7 +5820,7 @@
             "NodeImage"
           ],
           "x-ms-enum": {
-            "name": "NodeOSUpgradeChannel",
+            "name": "nodeOSUpgradeChannel",
```

Comment:
It was `NodeOSUpgradeChannel` in previous preview versions, but is `nodeOSUpgradeChannel` in GA versions. 

AFAIK, changing the case of the first letter of an enum type name in the Swagger has no impact on the generated type names, because each language breaks these up into words and then correctly cases them for the language in question?

We could fix the GA version to be Uppercase... but I guess this is fine.